apps/blockchain/lib/blockchain/transaction/receipt.ex

```diff
@@ -25,6 +40,34 @@ defmodule Blockchain.Transaction.Receipt do
           logs: EVM.SubState.logs()
         }
 
+  @spec new(EVM.trie_root(), EVM.Gas.t(), EVM.SubState.logs()) :: t()
+  def new(state_root, gas_used, logs) do
+    bloom_filter =
+      logs
+      |> Enum.reduce(Bloom.empty(), fn log_entry, acc ->
```

Comment:
Is this something that could reside within `Bloom` itself? Maybe something like` Bloom.from_logs(logs)`?